### PR TITLE
Implement signature field on Block struct

### PR DESCRIPTION
**Pull request by HarleyAppleChoi**
Closes: #250 

## Description

This implements the missing `signatures` field at [block](https://github.com/onflow/flow-go-sdk/blob/master/CONTRIBUTING.md#styleguides) struct at the FLOW api.

______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-go-sdk/blob/master/CONTRIBUTING.md#styleguides).
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: test/entities.go [hunk 1]
```diff
@@ -146,6 +149,7 @@ func (g *Blocks) New() *flow.Block {
 	return &flow.Block{
 		BlockHeader:  header,
 		BlockPayload: payload,
+		Signatures:   signiture,
```
Comment (sideninja) [inline test/entities.go:152]:
```suggestion
		Signatures:   signature,
```

File: test/entities.go [hunk 2]
```diff
@@ -146,6 +149,7 @@ func (g *Blocks) New() *flow.Block {
 	return &flow.Block{
```
Comment (sideninja) [inline test/entities.go:149]: It would be good to actually add a test that checks the signature presence.

**Review by sideninja — APPROVED** [review]
Good find, thanks :)

File: test/entities.go (hunk 2 above)
Comment (HarleyAppleChoi) [reply test/entities.go:150]: I added the signature struct on test/entity.go and assertion on client/client.test.go Is that what you mean?

Comment (sideninja) [reply test/entities.go:150]: Yes perfect, although I don't see the client test, did you push it?

Comment (HarleyAppleChoi) [reply test/entities.go:150]: Sorry. My bad. I think client_test.go does not need to be updated.

Comment (sideninja) [reply test/entities.go:150]: Good, then I would consider this done and approved. Thank you

**Pull request merged**